test/models/category_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CategoryTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Please write the necessary tests (usually using RSpec).

Reply:
Done. 04b5d52629254b8be02ccd0d3114a691daa84d5c